docs/extend/index.md

```diff
@@ -150,7 +150,7 @@ Consider the following `config.json` file.
 {
 	"description": "sshFS plugin for Docker",
 	"documentation": "https://docs.docker.com/engine/extend/plugins/",
-	"entrypoint": ["/go/bin/docker-volume-sshfs"],
+	"entrypoint": ["/docker-volume-sshfs"],
```

Comment:
I'm not sure this PR was opened by mistake or real pull request, but if this is a real PR, 

- Please update L168 as well
- Please remove TEST.md and update the description